Dockerfile

```
@@ -1,11 +1,11 @@
-FROM mcr.microsoft.com/dotnet/sdk:6.0.300-alpine3.15 AS build
+FROM mcr.microsoft.com/dotnet/sdk:6.0.400-alpine3.15 AS build
 WORKDIR /workspace/
 
 COPY src .
 
 RUN dotnet publish Altinn.AppTexts/Altinn.AppTexts.csproj -c Release -o /app_output
 
-FROM mcr.microsoft.com/dotnet/aspnet:6.0.5-alpine3.15 AS final
+FROM mcr.microsoft.com/dotnet/aspnet:6.0.8-alpine3.15 AS final
```

Lets upgrade to the latest and greatest
```suggestion
FROM mcr.microsoft.com/dotnet/aspnet:6.0.9-alpine3.16 AS final
```